src/main/java/com/foundation/view/AdvancedPanel.java

```diff
@@ -0,0 +1,271 @@
+        ownerField = new JTextField(15);
+        fileSizeLabel = new JLabel("File Size: ");
+        fileCompareList = new JComboBox();
+        fileSizeList = new JComboBox();
+        fileSizeField = new JTextField(10);
+        dateCreatedLabel = new JLabel("Date Created");
+        dateCreatedPicker = new JComboBox();
+        dateModifiedLabel = new JLabel("Date Modified");
+        dateModifiedPicker = new JComboBox();
+        dateAccessedLabel = new JLabel("Date Accessed");
+        dateAccessedPicker = new JComboBox();
+
+        DefaultComboBoxModel extModel = new DefaultComboBoxModel();
+        extModel.addElement(new FileCategory(0, "exe"));
+        extModel.addElement(new FileCategory(1, "txt"));
+        extModel.addElement(new FileCategory(2, "java"));
+        extModel.addElement(new FileCategory(3, "Other"));
+        extList.setModel(extModel);
+
+        extList.setPreferredSize(new Dimension(100, 20));
+        extList.setBorder(BorderFactory.createEtchedBorder());
+        extList.setSelectedIndex(1);
+
+        DefaultComboBoxModel visibilityModel = new DefaultComboBoxModel();
+        visibilityModel.addElement(new FileCategory(0, "Public"));
+        visibilityModel.addElement(new FileCategory(1, "Hidden"));
+        visibilityModel.addElement(new FileCategory(2, "All"));
+        visibilityList.setModel(visibilityModel);
+
+        visibilityList.setPreferredSize(new Dimension(100, 20));
+        visibilityList.setBorder(BorderFactory.createEtchedBorder());
+        visibilityList.setSelectedIndex(1);
+
+        DefaultComboBoxModel fileCompareModel = new DefaultComboBoxModel();
+        fileCompareModel.addElement(new FileCategory(0, "Less than"));
+        fileCompareModel.addElement(new FileCategory(1, "equals to"));
+        fileCompareModel.addElement(new FileCategory(2, "More than"));
+        fileCompareList.setModel(fileCompareModel);
+
+        fileCompareList.setPreferredSize(new Dimension(100, 20));
+        fileCompareList.setBorder(BorderFactory.createEtchedBorder());
+        fileCompareList.setSelectedIndex(1);
+
+        DefaultComboBoxModel fileSizeModel = new DefaultComboBoxModel();
+        fileSizeModel.addElement(new FileCategory(0, "Less than"));
+        fileSizeModel.addElement(new FileCategory(1, "equals to"));
+        fileSizeModel.addElement(new FileCategory(2, "More than"));
+        fileSizeList.setModel(fileSizeModel);
+
+        fileSizeList.setPreferredSize(new Dimension(100, 20));
+        fileSizeList.setBorder(BorderFactory.createEtchedBorder());
+        fileSizeList.setSelectedIndex(1);
+
+        Border innerBorder = BorderFactory.createTitledBorder("Advanced Search");
+        Border outerBorder = BorderFactory.createEmptyBorder(3, 3, 3, 3);
+        setBorder(BorderFactory.createCompoundBorder(outerBorder, innerBorder));
+
+        layoutComponents();
+    }
+
+    /*
+     * Form listener method
+     *
+     * @Author Ruben Mendoza
+     * @Version 1.0
+     */
+
```

Comment:
remove this blank line between the comment and method, remove on all places you added it.

Reply:
This class will not be used at this point; its content will change and the fixes will be included when it will be implemented